### PR TITLE
[Merged by Bors] - fix(FieldTheory/IntermediateField/Basic): fix typos in comment

### PR DESCRIPTION
**Pull request by kbuzzard**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/FieldTheory/IntermediateField/Basic.lean [hunk 1]
```diff
@@ -323,9 +323,11 @@ instance smulCommClass_right [SMul X Y] [SMul L Y] [SMulCommClass X L Y]
     (F : IntermediateField K L) : SMulCommClass X F Y :=
   inferInstanceAs (SMulCommClass X F.toSubfield Y)
 
---note: setting this istance the default priority may trigger trouble to synthize instance
---for field extension with more than one intermedaite field, example : in a field extension `F/E`,
---`K₁ ≤ K₂` are of type `intermediatefield F E`, the instance `IsScalarTower K₁ K₂ E`
+--note: giving this instance the default priority may trigger trouble with synthesizing instances
+--for field extensions with more than one intermediate field. For example, in a field extension
```
Comment (bryangingechen) [inline Mathlib/FieldTheory/IntermediateField/Basic.lean:327]:
```suggestion
-- note: giving this instance the default priority may trigger trouble with synthesizing instances
-- for field extensions with more than one intermediate field. For example, in a field extension
```

**Comment by bryangingechen** [comment]
Thanks!
bors d+

**Comment by kim-em** [comment]
bors merge

**Comment by kbuzzard** [comment]
Thanks. Sorry, I'm travelling.

**Pull request closed without merging**